content/en/docs/reference/config/analysis/ist0102/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: NamespaceNotInjected
+layout: analysis-message
+---
+
+This message occurs when you have a namespace that is missing the required
+annotation to indicate whether the namespace is auto-injected.
```

Comment:
can we show what the annotation is and how to set it, or link to another doc?

Reply:
fixed